app/web/src/components/ModelingDiagram/DiagramGroup.vue

```diff
@@ -427,8 +427,8 @@ const isAdded = computed(() => props.group.def.changeStatus === "added");
 
 const childCount = computed(() => {
   const mappedChildren = _.map(
-    props.group.def.childNodeIds,
-    (child) => useComponentsStore().componentsByNodeId[child],
+    props.group.def.childComponentIds,
+    (child) => useComponentsStore().componentsById[child],
   );
```

Comment:
I'll refrain from leaving many comments like this, but the `mappedChildren` abstraction is an example of many that are no longer needed because `Node <--> Component` no longer exists. We will fix abstractions like these as we begin to use them.